src/tess_asteroids/movingtpf.py

```diff
@@ -2708,6 +2746,25 @@ def _make_lc_hdulist(self):
                 if "aperture" in self.lc
                 else np.full(len(self.time), np.nan),
             ),
+            # TESS magnitude and error
+            fits.Column(
+                name="TESSMAG",
+                format="D",
```

Comment:
small thing, but using float precision should be enough for the magnitude values, same for the errors. We don't need double presition